attach/text_segment_transformer/agent-transformer.cpp

```diff
@@ -73,8 +73,7 @@ extern "C" void bpftime_agent_main(const gchar *data, gboolean *stay_resident)
 	cs_arch_register_x86();
 	bpftime::setup_syscall_tracer();
 	SPDLOG_DEBUG("Loading dynamic library..");
-	auto next_handle =
-		dlmopen(LM_ID_NEWLM, agent_so, RTLD_NOW | RTLD_LOCAL);
+	auto next_handle = dlmopen(LM_ID_NEWLM, agent_so, RTLD_NOW | RTLD_LOCAL);
```

Comment:
Is this a formatting change only?

Reply:
i had added APPLE specific code before, but since I had removed the `add_subdirectory` for this folder, I removed the guards, what remain was the formatting change.